sample/app/build.gradle

```diff
@@ -0,0 +1,156 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
```

Comment:
We have to be careful here, because the plugin needs to work with older versions as well. 
We'd need some proper testing setup for all the versions we support. 

Same goes for the Gradle version. Which one is it for the sample now?